site/docs/connectors/runtime/open-lineage-janus-connector.md

```diff
@@ -0,0 +1,90 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
+
+# Open Lineage Janus Connector
+
+The Open Lineage Janus connector allows the Open Lineages Services to connect with a JanusGraph database.
```

Comment:
Typo in Open _Lineages_ Services and link to Open Lineage Services page (/services/open-lineage-services/) + updates on that page to point back to the new connector page from the text "Lineage Janus Connector"